cpu/cortexm_common/Makefile.include

```diff
@@ -76,10 +88,11 @@ ifneq (,$(filter usbus_dfu tinyusb_dfu,$(USEMODULE)))
 else
   RIOTBOOT_LEN ?= 0x1000
 endif
+
 # Currently 2 slots are supported by default, equals in length
 NUM_SLOTS ?= 2
 # Take the whole flash minus RIOTBOOT_LEN and divide it by NUM_SLOTS
-SLOT0_LEN ?= $(shell printf "0x%x" $$((($(ROM_LEN:%K=%*1024)-$(RIOTBOOT_LEN)) / $(NUM_SLOTS))))
+SLOT0_LEN ?= $(shell printf "0x%x" $$((($(ROM_LEN:%K=%*1024)-$(RIOTBOOT_LEN)-$(SLOT_AUX_LEN)) / $(NUM_SLOTS))))
```

Comment:
This however would have to be moved to `sys/riotboot` with PR #19411. We can merge this PR first, I change it then in PR #19411.